docs/site/themes/template/layouts/partials/hero.html

```diff
@@ -2,11 +2,11 @@
 	<div class="wrapper">
 		<div class="text-block">
 			<h1 class="text-color-navy-blue">The Ultra-Fast Key-Value Store</h1>
-			<p>SplinterDB is an open-source embedded high-performance general-purpose key-value store built from the ground up to maximize the utilization of the latest generation of hardware</p>
+			<p>SplinterDB is an open source embedded high-performance general-purpose key-value store built from the ground up to maximize the utilization of the latest generation of hardware</p>
 			<div class="buttons">
                 <a class="button navy-blue" href="https://github.com/vmware/splinterdb/blob/main/docs/usage.md" target="_blank">Try it out</a>
                 <a class="button navy-blue" href="https://github.com/vmware/splinterdb">Github</a>
             </div>
 		</div>
 	</div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
I did not make this change, I assume this diff is because I did this in osx instead of windows.
Anyhow, no harm done.